.github/workflows/nightly-tests.yml

```diff
@@ -46,5 +46,5 @@ jobs:
       - name: Merge
         uses: actions/upload-artifact/merge@v4
         with:
-          name: cluster_test_logs
+          name: cluster_test_logs-${{ github.run_id }}-${{ github.job }}
```

Comment:
`test-cluster` and `merge-cluster-logs` are different jobs, is `github.job` the right context attribute to use? I think you might want to base the artifact name off of the test globs instead

Reply:
Oh, I see what you mean. I misread `job_id` as a unique ID like run_id, but yeah... glob makes more sense. I will update.